lib/contracts/contracts.js

```diff
@@ -215,7 +215,7 @@ class ContractsManager {
           contract = self.contracts[className];
 
           // look in code for dependencies
-          let libMatches = (contract.code.match(/\:(.*?)(?=_)/g) || []);
+          let libMatches = (contract.code.match(/:(.*?)(?=_)/g) || []);
```

Comment:
the `\` in `\:` is just not necessary or it wasn't catching some case before because of it?

Reply:
Just not necessary. Webstorm highlighted it for me and it was annoying so I removed it 🤷‍♂️ .